torchao/prototype/mxfp8_cuda/mxfp8_quantize.cuh

```diff
@@ -0,0 +1,1058 @@
+/*************************************************************************
+ * Copyright (c) 2022-2025, NVIDIA CORPORATION & AFFILIATES. All rights
+ *reserved.
+ *
+ * See LICENSE for license information.
```

Comment:
should there be a `LICENSE` file?

Reply:
I updated license headers per conversation with Supriya, see update. There's no standalone license file though.